### PR TITLE
procees 1 et 2

### PR DESCRIPTION
**Pull request by noeblp**

### CODE REVIEW

File: UI/processor.c [hunk 1]
```diff
@@ -51,4 +51,25 @@ void run_pretreatment(SDL_Surface *surface, int set, int angle){
...
     DET_All("../data/post_PRT.png", "../data/post_DET.png");
+}
+
+
+/**
+ * This function run the detection function according to the image out of pre-treatment (auto detect)
+ * @param int
+ * @return VOID
+ */
+void run_neural(int set)
```
Comment (awambst) [inline UI/processor.c:62]: This needs to be fixed, duplicate

**Review by awambst — CHANGES_REQUESTED** [review]
Fix proccessor.c

File: UI/processor.c (hunk 1 above)
Comment (awambst) [reply UI/processor.c:62]: And use or removed param

**Comment by sahamone** [comment]
Not good

**Pull request closed without merging**